src/EFCore.PG/Metadata/Internal/NpgsqlAnnotationNames.cs

```diff
@@ -24,6 +24,7 @@ public static class NpgsqlAnnotationNames
         public const string StorageParameterPrefix = Prefix + "StorageParameter:";
         public const string UnloggedTable = Prefix + "UnloggedTable";
         public const string IdentityOptions = Prefix + "IdentitySequenceOptions";
+        public const string TransformToTsVector = Prefix + "TransformToTsVector";
```

Comment:
Please rename to IndexToTsVector and group it with the other index annotations above

Reply:
Done.